test/cmd.js

```diff
@@ -1,4 +1,5 @@
 
+require("babel-core/register");
```

Comment:
Hmm I see what you mean, good suggestion. Ok, so the issue is that when the project gets generated with `--es6` the templates are changed to make use of that syntax. So in the tests then, if they're importing the files generated with `--es6` to run the tests, there's going to be a syntax error unless that require statement is present. 

I'll have to think some more about how to do this, I'm open to suggestions though. One idea would be put the ES6 test code in a separate file, which could have the require statement from above, but I don't know if that breaks from what you want the tests to look like.